docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,241 @@
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "pipeline": [
+    {
+      "type": "classifications",
+      "attributes": {
+        "classifier_name": "security",
+        "confidence_threshold": 0.5
+      }
+    }
+  ],
+  "source": "my-webcam"
+}
+```
+
+Note that the camera name in `"source"` and the `"classifier_name"` match the names that we configured for our source camera and verification system service, respectively.
+If you used different names, edit as applicable.
+View these [instructions to configure a transform camera](/components/camera/transform/) for reference.
+Save your config.
+
+## View your verification system in action
+
+Navigate to the **Control** tab.
+Expand the card matching the name of the transform camera you configured to view an image stream with the system's notifications overlaid.
+
+## How the verification system works
+
+The module sets up a state machine with 5 states:
+
+1. `TRIGGER_1`: The module begins in this state.
+   It is meant to be attached to a coarse, fast detector, like a simple motion detector.
+   This state runs the `trigger_1_detector` on every frame, looking for detections with any label from `trigger_1_labels` with at least `trigger_1_confidence`.
+   If the detector triggers, then the state moves to `TRIGGER_2`.
+   If no `TRIGGER_1` detector was specified in the config, the module moves immediately to state `TRIGGER_2`.
+2. `TRIGGER_2`: This state runs the `trigger_2_detector` on every frame, looking for detections with any label from `trigger_2_labels` with at least `trigger_2_confidence`.
+   If the detector triggers, then the state moves to `COUNTDOWN`.
+   If it doesn't trigger in 10 frames, it returns to state `TRIGGER_1`.
+3. `COUNTDOWN`: This state runs the `verification_detector` on every frame, looking for detections with any label from `verification_labels` with at least `verification_confidence`.
+   If the detector triggers, then the state moves to `DISARMED`.
+   If it doesn't trigger in the time specified by `countdown_time_s`, it moves to state `ALARM`.
+4. `ALARM`: The alarm state.
+   The module will emit the `ALARM` classification for the amount of time specified in `alarm_time_s`.
+   After that amount of time elapses, the module will return to state `TRIGGER_1`.
+5. `DISARMED`: The disarmed state.
+   The module will emit the `DISARMED` classification for the amount of time specified in `disarmed_time_s`.
+   After that amount of time elapses, the module will return to state `TRIGGER_1`.
+
+If you do not want the `ALARM` capabilities, and would like to just use it as a notification system when a detector gets triggered, you can set `disable_alarm: true` in the config, which will prevent `TRIGGER_2` from entering into the `COUNTDOWN` state.
+This means the system will only cycle between the states of `TRIGGER_1` and `TRIGGER_2`.
+
+You can use entering into the state `TRIGGER_2` as a way to send notifications.
```

Comment:
What does it currently? What does the alarm state do? I suppose nothing, but that's not made clear and left for the reader to guess about.

Reply:
What trigger 2 state does currently is described above, is that ok? Alarm is emitting classification. I can add in another sentence to make it clear that no sound or anything is triggered